Dockerfile

```diff
@@ -62,11 +62,25 @@ RUN apt-get update \
   && apt-get -y --no-install-recommends install \
 		build-essential \
 		libpq-dev \
-		libqpdf-dev \
-	&& python3 -m pip install --upgrade pip setuptools wheel \
+		git \
+		zlib1g-dev \
+		libjpeg62-turbo-dev \
+	&& if [ "$(uname -m)" = "armv7l" ]; \
+	  then echo "Building qpdf" \
+	  && mkdir -p /usr/src/qpdf \
+	  && cd /usr/src/qpdf \
+	  && git clone https://github.com/qpdf/qpdf.git . \
```

Comment:
Just a thought here, would it be better to switch to a released version instead of `main`?

ie `git checkout --quiet release-qpdf-10.6.2` following the clone

Reply:
Indeed, thanks!